pallets/rmrk-equip/src/mock.rs

```diff
@@ -40,21 +40,27 @@ frame_support::construct_runtime!(
 );
 
 parameter_types! {
-	pub const MaxPartsPerBase: u32 = 5;
+	// no longer needed as we use PartsLimit on the BoundedVec
+	// pub const MaxPartsPerBase: u32 = 5;
 	pub const MaxPropertiesPerTheme: u32 = 5;
+	pub const MaxCollectionsEquippablePerPart: u32 = 10;
 }
 
 impl pallet_rmrk_equip::Config for Test {
 	type Event = Event;
-	type MaxPartsPerBase = MaxPartsPerBase;
+	// no longer needed as we use PartsLimit on the BoundedVec
+	// type MaxPartsPerBase = MaxPartsPerBase;
 	type MaxPropertiesPerTheme = MaxPropertiesPerTheme;
+	type MaxCollectionsEquippablePerPart = MaxCollectionsEquippablePerPart;
 }
 
 parameter_types! {
 	pub ClassBondAmount: Balance = 100;
 	pub MaxMetadataLength: u32 = 256;
 	pub const MaxRecursions: u32 = 10;
 	pub const ResourceSymbolLimit: u32 = 10;
+	pub const PartsLimit: u32 = 10;
```

Comment:
only 10 parts? I think should be more allowed. Without proper benchmarking hard to tell but need some extra buffer ( how about 30 - 50 ) ?

Reply:
changing to 50.  didn't really apply any logic to the choice of number (either here or in runtime) just had to pick something.

Reply:
[196c9d8](https://github.com/rmrk-team/rmrk-substrate/pull/95/commits/196c9d8165e37bc0e2c93d132f456a0f049ea5f4)